client/my-sites/media-library/test/data-source.jsx

```diff
@@ -12,7 +12,7 @@ const noop = () => {};
 // we need to check the correct children are rendered, so this mocks the
 // PopoverMenu component with one that simply renders the children
 jest.mock( 'calypso/components/popover-menu', () => {
-	return ( props ) => <div>{ props.children }</div>;
+	return ( props ) => <div data-test={ props[ 'data-source' ] }>{ props.children }</div>;
```

Comment:
Do you mind using destructuring of arguments to avoid props object access like this? Something like:

```suggestion
	return ( { 'data-source': dataSource, children } ) => <div data-test={ dataSource }>{ children }</div>;
```